package.json

```diff
@@ -184,7 +184,8 @@
 		"react-dom": "^18.2.0",
 		"react-error-boundary": "^3.1.4",
 		"react-hot-toast": "^2.2.0",
-		"react-instantsearch-hooks-web": "^6.47.3",
+		"react-instantsearch": "^7.7.0",
+		"react-instantsearch-core": "^7.7.0",
```

Comment:
Since `react-instantsearch-core` is a dependency of `react-instantsearch`, and we don't import anything from `core` we can remove it from our dependencies.

Reply:
Good catch -- I was going off of the deprecation warning 🙈 
Fixed in https://github.com/hashicorp/dev-portal/pull/2368/commits/02e9194717869e1e17c701eef42ab8781498097e